avocado_vt/test.py

```diff
@@ -395,17 +395,14 @@ def _runTest(self):
             test_modules[t_type] = imp.load_module(t_type, f, p, d)
             f.close()
 
-        # TODO: the environment file is deprecated code, and should be removed
-        # in future versions. Right now, it's being created on an Avocado temp
-        # dir that is only persisted during the runtime of one job, which is
-        # different from the original idea of the environment file (which was
-        # persist information accross virt-test/avocado-vt job runs)
+        # Open the environment file
         env_filename = os.path.join(data_dir.get_tmp_dir(),
                                     params.get("env", "env"))
         env = utils_env.Env(env_filename, self.env_version)
-        self.runner_queue.put({"func_at_exit": cleanup_env,
-                               "args": (env_filename, self.env_version),
-                               "once": True})
+        if params.get_boolean("job_env_cleanup", "no"):
```

Comment:
The default value should be `"yes"` as normally we always want to do env_cleanup.

Reply:
I think some of the changes got lost from too many iterations of the PR. Fixed now.